util/expensivequery/expensivequery.go

```diff
@@ -78,11 +78,6 @@ func (eqh *Handle) Run() {
 	}
 }
 
-// Close closes the handle and release the background goroutine.
```

Comment:
Why remove this? 

em... Actually, this function is useless, but, It ok to not close  `exitCh `?

Reply:
`extiCh` is closed in both `domain.Close` and `expensiveQueryHandl.Close`, lead to panic. @crazycs520

Reply:
/cc @XuHuaiyu